docs/core/resilience/http-resilience.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Resilience-patterns for HTTP apps
+description: 
+author: IEvangelist
+ms.author: dapine
+ms.date: 09/28/2023
+---
+
+# Resilience-patterns for HTTP apps
+
+Building robust HTTP apps that can recover from transient fault errors is a common requirement. To help build resilient HTTP apps, the [Microsoft.Extensions.Http.Resilience](https://www.nuget.org/packages/Microsoft.Extensions.Http.Resilience) NuGet package provides resilience mechanisms specifically for the <xref:System.Net.Http.HttpClient>. This NuGet package is built on top of _Polly_, which is a very popular open-source project. For more information, see [Polly](https://github.com/App-vNext/Polly).
+
+## Get started
+
+To use resilience-patterns in HTTP apps, install the [Microsoft.Extensions.Http.Resilience](https://www.nuget.org/packages/Microsoft.Extensions.Http.Resilience) NuGet package.
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.Http.Resilience --version 8.0.0
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<PackageReference Include="Microsoft.Extensions.Http.Resilience" Version="8.0.0" />
+```
+
+---
+
+## Add resilience handlers to an HTTP client
+
+To add resilience to an <xref:System.Net.Http.HttpClient>, you chain a call on the <xref:Microsoft.Extensions.DependencyInjection.IHttpClientBuilder> type which is returned from calling any of the available <xref:Microsoft.Extensions.DependencyInjection.HttpClientFactoryServiceCollectionExtensions.AddHttpClient%2A> methods. There are several resilience-centric extensions available, some are standard employing various industry best practices, and others are more customizable.
+
+### The standard resilience handler
+
+The standard resilience handler uses multiple resilience strategies with default options to send the requests and handle any transient errors. The standard resilience handler is added by calling the `AddStandardResilienceHandler` extension method.
+
+```csharp
+// A service collection is manually instantiated for this example.
+// If you're using a generic host, you'll instead use its service collection:
+//
+// var host = Host.CreateApplicationBuilder(args);
+// host.Services.AddHttpClient<ExampleClient>(client => { ... });
+
+var services = new ServiceCollection();
+
+services.AddHttpClient<ExampleClient>(client =>
```

Comment:
capture the result here i.e. `IStandardResilienceHandlerBuilder` and note that it contains addiotional extensions to configure the handler.